python/lsst/ip/isr/isrTask.py

```diff
@@ -597,7 +599,16 @@ def getIsrExposure(self, dataRef, datasetType, immediate=True):
         try:
             exp = dataRef.get(datasetType, immediate=immediate)
         except Exception, e:
-            raise RuntimeError("Unable to retrieve %s for %s: %s" % (datasetType, dataRef.dataId, e))
+            try:
+                if not self.config.fallbackFilterName:
+                    raise ValueError("No fallback filter specified")
+                exp = dataRef.get(datasetType, filter=self.config.fallbackFilterName, immediate=immediate)
+                self.log.warn("Using fallback calibration from filter %s" % self.config.fallbackFilterName)
+            except ValueError as e:
+                raise RuntimeError("Unable to retrieve %s for %2: %s and not fallbackFilterName defined" %\
+                        (datasetType, dataRef.dataId, e))
+            except:
+                raise RuntimeError("Unable to retrieve %s for %s: %s" % (datasetType, dataRef.dataId, e))
```

Comment:
One was intended to let the user know that something had failed (maybe that dataref was  bad) and the other was intended to inform the user that the failure was specifically may be fixed simply by supplying a fallback filter. I will think about this a bit and see if another way comes to me.